deeppavlov/core/commands/train.py

```diff
@@ -154,7 +155,7 @@ def train_evaluate_model_from_config(config_path: str, is_trained=False) -> None
 
     metrics_functions = list(zip(train_config['metrics'], get_metrics_by_names(train_config['metrics'])))
 
-    if not is_trained:
+    if train_config['train_model']:
```

Comment:
It should not be necessary to change config file to evaluate the model without training

Reply:
Now `train`, `validate` and 'test' flags are used analogously, I see no reason to set one of them in code and another in config. Either we have three optional parameters of the function called `train_model`, `evaluate_model` and `test_model`, or we have three fields defined in config. Probably,  both modes can be allowed.

Reply:
By the way, now the code is more concise than it was.